### PR TITLE
Edits addressing changes in abstractions 5

### PR DESCRIPTION
**Pull request by westin-m**
Changes that address breaking changes in Abstractions 5

### CODE REVIEW

**Review by Exantos — APPROVED** [review]

File: src/Microsoft.Identity.Web.GraphServiceClient/GraphAuthenticationProvider.cs [hunk 1]
```diff
@@ -19,9 +19,7 @@ namespace Microsoft.Identity.Web
     /// </summary>
     internal class GraphAuthenticationProvider : IAuthenticationProvider
     {
-        const string ScopeKey = "scopes";
```
Comment (jmprieur) [inline src/Microsoft.Identity.Web.GraphServiceClient/GraphAuthenticationProvider.cs:22]: Is it unrelated

File: src/Microsoft.Identity.Web.GraphServiceClient/GraphAuthenticationProvider.cs [hunk 2]
```diff
@@ -19,9 +19,7 @@ namespace Microsoft.Identity.Web
...
         private const string AuthorizationHeaderKey = "Authorization";
-        private const string AuthorizationHeaderProviderOptionsKey = "authorizationHeaderProviderOptions";
```
Comment (jmprieur) [inline src/Microsoft.Identity.Web.GraphServiceClient/GraphAuthenticationProvider.cs:24]: Is it unrelated

**Review by jmprieur — APPROVED** [review]

**Review by JoshLozensky — APPROVED** [review]
lgtm

**Review by jennyf19 — APPROVED** [review]
:shipit:

File: src/Microsoft.Identity.Web.GraphServiceClient/GraphAuthenticationProvider.cs (hunk 2 above)
Comment (westin-m) [reply src/Microsoft.Identity.Web.GraphServiceClient/GraphAuthenticationProvider.cs:24]: they are unused

**Pull request merged**